### DISCUSSION TITLE
OpenVoiceOs setup

### DISCUSSION THREAD

**Discussion by Vullturee** [OpenVoiceOS/ovos-buildroot#201] [Q&A] [answered]
How can we install  OpenVoiceOs on raspberry pi 4b from scratch?
Can you explain in detail?

**Comment by JarbasAl** [comment, marked as answer]
easiest way would be with https://github.com/OpenVoiceOS/ovos-installer

**Reply by Vullturee** [reply]
I did not understand but?

**Reply by Vullturee** [reply]
Should I write this file to SD card?

**Reply by goldyfruit** [reply]
You just need to burn a standard Raspberry Pi OS on the SD card via RPi Imager, then SSH on the Raspberry Pi and run the `ovos-installer`.
